src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -629,15 +629,13 @@
   [message-content-wrapper message
    [unknown-content-type message]])
 
-(defn chat-message [{:keys [display-photo? pinned pinned-by] :as message} space-keeper]
+(defn chat-message [{:keys [display-photo? pinned pinned-by] :as message}]
   [:<>
    [reactions/with-reaction-picker
     {:message         message
      :reactions       @(re-frame/subscribe [:chats/message-reactions (:message-id message) (:chat-id message)])
-     :picker-on-open  (fn []
-                        (space-keeper true))
-     :picker-on-close (fn []
-                        (space-keeper false))
+     :picker-on-open  (fn [])
+     :picker-on-close (fn [])
```

Comment:
Can be replace with `comment` which will always return `nil`
`:picker-on-open comment`